README.md

```diff
@@ -37,3 +37,4 @@ $ npm start
 
 ## Developer ⚒️
 [Enmanuel Lassis](https://github.com/elassis)
```

Comment:
- **[OPTIONAL]** Instead of linking to your Github profile only, it would be nice to link to your other professional social media handles like Twitter and LinkedIn